rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -255,22 +255,28 @@ namespace rsx
 
 	void thread::capture_frame(const std::string &name)
 	{
-		frame_capture_data::draw_state draw_state = {};
+		frame_trace_data::draw_state draw_state = {};
 
-		int clip_w = rsx::method_registers.surface_clip_width();
+		/*int clip_w = rsx::method_registers.surface_clip_width();
```

Comment:
You can remove this if you wish, its based on some old reasoning on what can invalidate a draw command and is technically incorrect/incomplete anyway.